IsraelHiking.Web/src/application/application.module.ts

```diff
@@ -356,3 +359,7 @@ export class ApplicationModule {
         angulartics2GoogleAnalytics.startTracking();
     }
 }
+
+export function playerFactory() {
```

Comment:
I suggest to simply inline this: `() => player` in the above `forRoot(...`)`.

Reply:
I'm leaving it to you (not sure how)